src/Membership/UnregistrationFactory.php

```diff
@@ -19,4 +19,17 @@ public function createFromUnregistrationCommandAndAdherent(UnregistrationCommand
 
         return $unregistration;
     }
+
+    public function createFromAdherent(Adherent $adherent): Unregistration
```

Comment:
This factory is not needed, since it has no deps, those methods could just be static and live in the `Unregistration` class.

Reply:
Done